web/src/app/projects/repository/repository.component.html

```diff
@@ -123,6 +123,14 @@ <h4 mat-line class="repository-detail-card__content__title repository-detail-car
                 <img matListAvatar [src]="pullRequest.owner.avatarUrl" [alt]="pullRequest.owner.username" />
                 <h4 mat-line class="repository-detail-card__content__title">{{ pullRequest.title }}</h4>
                 <p mat-line class="repository-detail-card__content__request-user">{{ pullRequest.owner.username }}</p>
+                <p mat-line class="repository-detail-card__content__request">Commits: {{ pullRequest.commits }}</p>
+                <p mat-line class="repository-detail-card__content__request">Comments: {{ pullRequest.comments }}</p>
```

Comment:
Found this defect with comments and raised it separately https://github.com/DashboardHub/PipelineDashboard/issues/1586